### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by januswel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+npm install @classmethod/lifft-up
+// or
+yarn add @classmethod/lifft-up
+```
+
+## Usage
+
+Authentication by IDToken.
+
+```
```
Comment (januswel) [inline README.md:25]: Add language hint 🙏🏼 
These codes are ... JavaScript?

File: README.md [hunk 2]
```diff
@@ -1 +1,38 @@
...
+import * as lifftUp from '@classmethod/lifft-up'
+
+(async () => {
+	// https://developers.line.biz/ja/reference/liff/#get-id-token
+	const idToken = liff.getIDToken()
+	// Specify Liff Channel ID.
+	const clientId = '1234567890'
+	// The IDToken payload will be returned.
+	const payload = await lifftUp.authorizer(idToken, clientId)
```
Comment (januswel) [inline README.md:34]: Change tab width to 2 🙇🏼

File: README.md [hunk 3]
```diff
@@ -1 +1,38 @@
...
+
+## Usage
+
+Authentication by IDToken.
+
+```
+import * as lifftUp from '@classmethod/lifft-up'
+
+(async () => {
+	// https://developers.line.biz/ja/reference/liff/#get-id-token
```
Comment (januswel) [inline README.md:29]: Change this reference URL to the one in English

**Review by januswel — COMMENTED** [review]
@ota-seiji 👍🏼

File: README.md (hunk 1 above)
Comment (ota-seiji) [reply README.md]: done

**Review by ota-seiji — APPROVED** [review]
LGTM!

File: README.md (hunk 2 above)
Comment (ota-seiji) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (ota-seiji) [reply README.md]: done

**Pull request merged**